packages/sn-filter-pane/src/hooks/use-render.ts

```diff
@@ -34,5 +32,5 @@ export default function useRender() {
     return (() => {
       teardown(root);
     });
-  }, [constraints, optionsString, listboxIdsString, resourcesReady]);
+  }, [constraints, listboxIdsString, resourcesReady]);
```

Comment:
Does it work to change order of listboxes, will it trigger a re-render? (Or what was the purpose of the `optionsString`).

Reply:
Yes, this was used to update options when options were passed from sense-client. Now they are passed via properties instead.